django/contrib/admindocs/utils.py

```diff
@@ -107,11 +107,9 @@ def parse_rst(text, default_reference_context, thing_being_parsed=None):
 
 
 def create_reference_role(rolename, urlbase):
-    def _role(name, rawtext, text, lineno, inliner, options=None, content=None):
```

Comment:
Do we need to follow deprecation policy for `content=None` kwarg?  I guess so. What do you think?

Reply:
I think you should leave it as it seems to be the docutils interface for these methods. The `if content is None:` block can go though.